src/Kmd.Logic.Gateway.Automation/ItemValidationResultBase.cs

```diff
@@ -2,6 +2,8 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
+using Newtonsoft.Json;
```

Comment:
are these namespace in use ?

Reply:
It is already fixed. 

![image](https://user-images.githubusercontent.com/12545474/103351786-4d8f0580-4aca-11eb-81b0-df0cfca66006.png)